src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -15,7 +14,7 @@ export const ErrorNotification: React.FC<Props> = ({ error, setIsError }) => {
     }, 3000);
   }, []);
 
-  const getErrorsNotification = useMemo(() => {
+  const getErrorNotification = useMemo(() => {
```

Comment:
for such a case (when you have some logic in your `memo` function) better use `useCallback` hook, and call the resulting function at the place

and when you use `useMemo` function you should just return some value you want to memorize (and the variable name should be without a verb, i.e `errorNotification`)

This comment is just FYI (no need to re-request review)